### PR TITLE
added faq entries about tags

### PR DESCRIPTION
**Pull request by DerMolly**
### Description
This PR adds a faq entry about tags (currently as a draft).

### Steps

- [x] Added changes
- [x] I read the [contribution documentation](https://github.com/hedgedoc/hedgedoc.github.io/blob/main/CONTRIBUTING.md) and signed-off my commits to accept the DCO.

### CODE REVIEW

File: content/english/faq.md [hunk 1]
```diff
@@ -60,6 +60,36 @@ Starting with HedgeDoc 2.0, mermaid is used for rendering sequence-diagrams. The
     ```
 {{< /faq-entry >}}
 
+{{< faq-entry title="Why did the way tags are set change?" draft=true >}}
```
Comment (mrdrogdrog) [inline content/english/faq.md:63]: I would split this into two questions because that are two different changes.

**Review by mrdrogdrog — CHANGES_REQUESTED** [review]

File: content/english/faq.md (hunk 1 above)
Comment (DerMolly) [reply content/english/faq.md]: done

File: content/english/faq.md [hunk 2]
```diff
@@ -60,14 +60,19 @@ Starting with HedgeDoc 2.0, mermaid is used for rendering sequence-diagrams. The
     ```
 {{< /faq-entry >}}
 
-{{< faq-entry title="Why did the way tags are set change?" draft=true >}}
+{{< faq-entry title="Why do tag headings don't work anymore?" draft=true >}}
```
Comment (mrdrogdrog) [inline content/english/faq.md:63]:
```suggestion
{{< faq-entry title="Why do tags in headings don't work anymore?" draft=true >}}
```

File: content/english/faq.md [hunk 3]
```diff
@@ -60,14 +60,19 @@ Starting with HedgeDoc 2.0, mermaid is used for rendering sequence-diagrams. The
...
 ```
 #### tags: `tag1`, `tag2`
 ```
 
-Also we deprecated the usage of comma-separated "lists". Instead you should use [YAML Collections](https://yaml.org/spec/1.2/spec.html#id2759963).
+Instead please use the [frontmatter metadata](https://demo.hedgedoc.org/yaml-metadata#tags) to specify tags.
+
+{{< /faq-entry >}}
+
+{{< faq-entry title="Why is the comma-separated definition of tags in the YAML-metadata deprecated?" draft=true >}}
```
Comment (mrdrogdrog) [inline content/english/faq.md:74]:
```suggestion
{{< faq-entry title="Why is the comma separated definition of tags in the YAML metadata deprecated?" draft=true >}}
```

File: content/english/faq.md [hunk 4]
```diff
@@ -60,14 +60,19 @@ Starting with HedgeDoc 2.0, mermaid is used for rendering sequence-diagrams. The
...
+We try to introduce as few hedgedoc-specific things as possible, especially if there is a normal way to do it anyway. With this in mind the old syntax seems unnecessary extra, instead you should use [YAML Collections](https://yaml.org/spec/1.2/spec.html#id2759963) to specify tags as this is standard way of providing list in YAML.
```
Comment (mrdrogdrog) [inline content/english/faq.md:75]: That sounds like an excuse.
Why don't we use something simple like `[YAML already has a definition for lists](https://yaml.org/spec/1.2/spec.html#id2759963). Therefore we don't see any reason to maintain another way to define a list.`

**Review by mrdrogdrog — CHANGES_REQUESTED** [review]

File: content/english/faq.md (hunk 3 above)
Comment (DerMolly) [reply content/english/faq.md:74]: this was copied from the translation of the other PR. Maybe we want to keep this consistent…

File: content/english/faq.md (hunk 2 above)
Comment (DerMolly) [reply content/english/faq.md]: done

File: content/english/faq.md (hunk 3 above)
Comment (DerMolly) [reply content/english/faq.md]: done

File: content/english/faq.md (hunk 4 above)
Comment (DerMolly) [reply content/english/faq.md]: done

**Review by mrdrogdrog — APPROVED** [review]

File: content/english/faq.md [hunk 5]
```diff
@@ -60,6 +60,41 @@ Starting with HedgeDoc 2.0, mermaid is used for rendering sequence-diagrams. The
...
 {{< /faq-entry >}}
 
+{{< faq-entry title="Why do tags in headings don't work anymore?" draft=true >}}
+Starting with HedgeDoc 2.0, we don't support tag headings in the note anymore. 
+
+```
+#### tags: `tag1`, `tag2`
+```
+
+Instead please use the [frontmatter metadata](https://demo.hedgedoc.org/yaml-metadata#tags) to specify tags.
```
Comment (ErikMichelson) [inline content/english/faq.md:70]: With version 2.0 this URL will contain an additional /n/.

Comment (DerMolly) [reply content/english/faq.md:70]: good point

**Review by ErikMichelson — APPROVED** [review]

**Pull request merged**